GenderPayGap.WebUI/Models/Organisation/OrganisationReportsViewModel.cs

```diff
@@ -17,29 +17,25 @@ public class OrganisationReportsViewModel
         public long OrganisationId { get; set; }
 
         [BindNever]
-        public Dictionary<int, OrganisationScopeForYear> OrganisationScopesByYear { get; set; }
+        public List<OrganisationScopeForYear> OrganisationScopesByYear { get; set; }
 
         public OrganisationReportsViewModel(Database.Organisation organisation, List<int> yearsWithDraftReturns)
         {
             this.OrganisationId = organisation.OrganisationId;
             this.OrganisationScopesByYear = GetOrganisationScopes(organisation, yearsWithDraftReturns);
         }
 
-        private Dictionary<int, OrganisationScopeForYear> GetOrganisationScopes(Database.Organisation organisation, List<int> yearsWithDraftReturns)
+        private List<OrganisationScopeForYear> GetOrganisationScopes(Database.Organisation organisation, List<int> yearsWithDraftReturns)
         {
-            // Get list of all available reporting years
             List<int> reportingYears = ReportingYearsHelper.GetReportingYears();
             
-            // Create dictionary to store reporting years as keys against OrganisationScopeForYear values
-            Dictionary<int, OrganisationScopeForYear> scopesByYear = new Dictionary<int, OrganisationScopeForYear>();
+            List<OrganisationScopeForYear> scopesByYear = new List<OrganisationScopeForYear>();
             
-            // Iterate over each available reporting year
             foreach (int reportingYear in reportingYears)

```

Comment:
Fun fact: you can do this in a super functional way using Linq (e.g. `reportingYears.Select(organisation.GetScopeForYear)`, filter on not-nulls, then Select to create the OrganisationScopeForYear) but this is probably clearer if we're not doing things the functional way elsewhere

Reply:
Ah yeah, I love these functional Linq extras!
I'm really keen on functional stuff, but aware that that's not classic C# style and might be a bit unfamiliar to people with a C# background picking up maintenance in future. Also aware that other places use `foreach`, so trying to keep it consistent! :)